packages/network-navigator/src/determineDomain.ts

```diff
@@ -27,25 +27,25 @@
 export const determineDomain: (
 	bilinks: any[],
 	fn: (i: any) => number,
-	configMin?: number,
-	configMax?: number,
+	configMin: number | null,
+	configMax: number | null,
 ) => [number, number] = (
 	bilinks: any[],
 	fn: (i: any) => number,
-	configMin?: number,
-	configMax?: number,
+	configMin: number | null,
+	configMax: number | null,
 ) => {
-	if (configMin !== undefined && configMax !== undefined) {
+	if (configMin !== null && configMax !== null) {
```

Comment:
With null/undefined checks, we shouldn't use triple-equals. Double-equals will check for null or undefined, triple-equals checks for exactly one.

if (configMin != null && configMax != null)